wsimod/orchestration/model.py

```diff
@@ -144,6 +144,22 @@ def __init__(self):
         # self.arcs_type = {} #not sure that this would be necessary
         self.nodes = {}
         self.nodes_type = {}
+        
+        # Default orchestration
+        self.orchestration = [{'FWTW' : 'treat_water'},
+                                {'Demand' : 'create_demand'},
+                                {'Land' : 'run'},
+                                {'Groundwater' : 'infiltrate'},
+                                {'Sewer' : 'make_discharge'},
+                                {'Foul' : 'make_discharge'},
+                                {'WWTW' : 'calculate_discharge'},
+                                {'Groundwater' : 'distribute'},
+                                {'River' : 'calculate_discharge'},
+                                {'Reservoir' : 'make_abstractions'},
+                                {'Land' : 'apply_irrigation'},
+                                {'WWTW' : 'make_discharge'},
+                                {'Catchment' : 'route'}]
```

Comment:
As discussed, this seems like the right approach, but I would try to think on a way of facilitating the user to use this proven and tested workflow without having to re-write it from scratch. In other words, setting up a function (happy to write it) that replaces all the `Growndwater` steps with `MyGroundwater` steps if I'm using a custom groundwater node.

Reply:
@liuly12 @barneydobson , any comment on this?

Reply:
Agree. I'm also wondering it might be worth printing this orchestration list (i.e., sequence of calculation order in the model) after the replacement, to let the users fully notice and reconfirm the change of calculation order. This is because the change of calculation order is often the source of bugs after adding new node types.